### ISSUE TITLE
Photo Gallery

### ISSUE THREAD

**Issue by jankincl** [vue-comps/vue-zoombox#3]
I decided to create a small "gallery" from your component. Basically I just need to add arrows over the opened image, and then add functions to them. I am having troubles adding a <div> in the template to wrap around  the <img class=zoombox-image >. 
If I understand correctly, you had option to add html content in the previous versions? Would you know how to make it work?

Or, even maybe, would you be interested in creating such a photo gallery component by yourself? I know it's a bit daring to suggest/ask that, but I searched a lot, and there isn't really any nicely working vuejs photo gallery component available yet.

**Comment by paulpflug** [comment]
Hey,

if you want to add arrows you can use the opened slot in v1.0.2:

``` html
<zoombox src="./path/to/img" thumb="./path/to/thumb">
  <span>A caption</span>
  <div slot="opened">
    <!-- your arrows -->
 </div>
</zoombox>
```

but you have to position them correctly yourself.

I'm thinking about a photo gallery, but currently I'm very busy.

**Comment by jankincl** [comment]
I ended up using similar computed style as you have for captions in the component, to position the arrows on the left and right side of the image. However I wasn't able to get the functions to work in the end I guess I would have to refer somehow the previous and next instances of zoombox in props and open them, but it's just above my programming skills, so I gave up for now. I will use simple zoombox for now and will wait patiently if you ever find the time to look into the photo gallery.

Thanks for the components!
